src/data_manager/collectors/tickets/integrations/jira.py

```diff
@@ -104,17 +107,28 @@ def _fetch_ticket_resources(self) -> Iterator[TicketResource]:
                 created_at=created_at or None,
                 metadata={k: v for k, v in metadata.items() if v},
             )
-
-            logger.debug(f"Collected JIRA ticket {issue_key}")
+            
+            if collect_since:
+                logger.debug(f"Ticket {issue_key} was recently pulled from client as it was updated or created after last collection date {collect_since.strftime('%Y-%m-%d %H:%M:%S')}")
```

Comment:
Can you update this log line as follows:
- https://github.com/haozturk/A2rchi/commit/02090c9329710afcb90932dcf98bdef28a47c283
It's more descriptive for debugging